docker-compose.yml

```diff
@@ -59,6 +59,21 @@ services:
       retries: 5
       start_period: 30s
 
+  json-server:
+    # image: "clue/json-server"
+    build:
+      dockerfile_inline: |
+        FROM node:alpine
+        ENV NODE_PATH=/usr/local/lib/node_modules/
+        RUN npm install -g json-server --save-dev
+        COPY ./data/user.js ./data/db.json /
+        EXPOSE 3000
+        ENTRYPOINT ["node", "user.js"]
+    container_name: "json_server"
+    ports:
+      - "3000:3000"
+    restart: always
```

Comment:
that's not an issue - just interesting why installed the server from the ground instead of using `image: "vimagick/json-server:latest"`?

Reply:
I've tried to launch one of a bunch available json-server images, but, there's an issue: they all install it as a command, while in order to use it in node's `require` you need to use `--save-dev` option and I've just made my own finally =)